include/gridtools/stencil/frontend/axis.hpp

```diff
@@ -51,7 +51,7 @@ namespace gridtools {
             static constexpr size_t n_intervals = NIntervals;
 
             using axis_interval_t =
-                core::interval<core::level<0, 1, LevelOffsetLimit>, core::level<NIntervals, -1, LevelOffsetLimit>>;
```

Comment:
Dycore stencils have to be checked if we can live without this. (It's the hard-coded "extra offsets around full interval".)